apps/resnet_50/Makefile

```diff
@@ -0,0 +1,31 @@
+include ../support/Makefile.inc
+
+BIN ?= bin
+SEED = 123
+
+all: $(BIN)/process
+
+$(BIN)/pytorch_weights/ok:
+	@mkdir -p $(@D)
+	python3 load_weights.py $(@D)
+	echo "ok" > $@
+
+$(BIN)/resnet50.generator: Resnet50Generator.cpp $(GENERATOR_DEPS)
+	@mkdir -p $(@D)
+	$(CXX) $(CXXFLAGS) -g -fno-rtti $(filter-out %.h,$^) -o $@ $(LDFLAGS) $(HALIDE_SYSTEM_LIBS)
+
+$(BIN)/resnet50.a: $(BIN)/resnet50.generator
+	@mkdir -p $(@D)
+	$^ -g resnet50 -o $(BIN) -f resnet50 target=$(HL_TARGET) auto_schedule=false
+
+$(BIN)/process: process.cpp $(BIN)/resnet50.a 
+	@mkdir -p $(@D)
+	$(CXX) $(CXXFLAGS) -I$(BIN) -Wall -O3 $^ -o $@ $(LDFLAGS) $(IMAGE_IO_FLAGS) $(CUDA_LDFLAGS) $(OPENCL_LDFLAGS) $(OPENGL_LDFLAGS)
+
+benchmark_and_validate: $(BIN)/process $(BIN)/pytorch_weights/ok
```

Comment:
We should add resnet50 to TEST_APPS in the top-level Makefile, and add a "test" target to this makefile that ensures the generator builds (i.e. just build bin/process).

It won't validate correctness (which would require all the pytorch deps, and downloading resnet weights), but we'll have this app as a compile-time canary for future PRs.

Reply:
Done! I added app/resnet_50 build to test_app in the last commit.